backend/package.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
let's setup Lerna to be able to manage dependencies between backend & frontend at one place

https://lerna.js.org/docs/getting-started#adding-lerna-to-an-existing-repo